includes/classGlidingDB.php

```diff
@@ -0,0 +1,170 @@
+<?php
+require_once dirname(__FILE__) . '/classSQLPlus.php';
+class GlidingDB extends SQLPlus 
```

Comment:
@tim-hogan where is the definition of `SQLPlus` ? does it come from a library or did you write it? I can not find it. Could it be that you forgot to add it to git?

Reply:
I forgot to load, now loaded